packages/tocco-ui/src/Menu/Item.js

```diff
@@ -0,0 +1,70 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import {stylingLook} from '../utilStyles'
+
+const StyledItem = styled.li`
+  position: relative;
+
+  > ul {
+    display: ${props => props.isOpen ? 'block' : 'none'}
+  }
+`
+
+class Item extends React.Component {
+  state = {
+    isOpen: this.props.isOpen
+  }
+
+  toogleOpenState = () => {
+    // eslint-disable-next-line
+    this.props.isToggleable
+    && this.setState(prevState => ({
```

Comment:
in "normal" JS (not JSX), i personally prefer "conventional" if statements:
```
if (this.props.isToggleable) {
  this.setState(...);
}
```

Reply:
IMHO I consider "short circuit" as best practise. It has exactly they same effect like "conventional" if statements but is shorter. I have no strong opinions about this or that way. May  let us discuss this with other JS developers?